docs/quickstart.md

```diff
@@ -1,19 +1,11 @@
 ---
-title: Quickstart
-excerpt: This page will help you get started with Phylum CLI tool. You'll be up and running in a jiffy!
-category: 6255e67693d5200013b1fa3e
-hidden: false
+# Make this a docs-only site, with this doc as the front page
+slug: /
 ---
 
-<p align="center">
-  <img height="100" src="https://raw.githubusercontent.com/phylum-dev/cli/main/assets/dark-bckg.svg">
-</p>
+# Quickstart
 
----
-
-# Introduction
-
-The command line interface (CLI) allows users to submit their project package dependencies to [Phylum's](https://phylum.io) API for analysis. Currently [pre-built binaries](https://github.com/phylum-dev/cli/releases) for Linux and macOS are available. On Windows, we recommend using the Linux binaries under [WSL](https://learn.microsoft.com/en-us/windows/wsl/). For more options, see [Alternate Installation Methods](https://docs.phylum.io/docs/alternate_install).
+The Phylum command line interface (CLI) allows users to submit their project package dependencies to [Phylum's](https://phylum.io) API for analysis. Currently [pre-built binaries](https://github.com/phylum-dev/cli/releases) for Linux and macOS are available. On Windows, we recommend using the Linux binaries under [WSL](https://learn.microsoft.com/en-us/windows/wsl/). For more options, see [Alternate Installation Methods](./alternate_install.md).
 
 [![asciicast](https://asciinema.org/a/431262.svg)](https://asciinema.org/a/431262)
```

Comment:
@andreaphylum provided feedback to suggest changing this animation so it is not so small and also so that it is current.

Reply:
The animation was removed entirely because it was outdated and provided no additional content/value over the text that is already on the Quickstart/README pages.